packages/@aws-cdk/toolkit/lib/api/cloud-assembly/private/source-builder.ts

```diff
@@ -9,6 +10,12 @@ import { ToolkitError } from '../../errors';
 import { debug } from '../../io/private';
 import { AssemblyBuilder, CdkAppSourceProps } from '../source-builder';
 
+// bypass loading from disk if we already have a supported object
+const CLOUD_ASSEMBLY_SYMBOL = Symbol.for('@aws-cdk/cx-api.CloudAssembly');
+function isCloudAssembly(x: any): x is cxapi.CloudAssembly {
```

Comment:
Why not `CloudAssembly.isCloudAssembly()` ?

Reply:
Good catch! Forgot to update this.

Reply:
updated.